src/main/java/com/github/jmchilton/blend4j/galaxy/HistoriesClient.java

```diff
@@ -37,6 +38,20 @@
   HistoryExport exportHistory(String historyId);
   
   /**
+   * Deletes the given History from Galaxy (this will not purge).
+   * @param historyId  The id of the History to delete.
+   * @return A {@link ClientResponse} for this request.
+   */
+  ClientResponse deleteHistoryResponse(String historyId);
```

Comment:
What's the difference between responding with a `ClientResponse` and a `HistoryDeleteResponse`? Why do you have both?

As an API user, how do I decide which one to use?